utilities/cli/holohub.py

```diff
@@ -336,6 +355,73 @@ def handle_run_container(self, args: argparse.Namespace) -> None:
             verbose=args.verbose,
         )
 
+    def handle_test(self, args: argparse.Namespace) -> None:
+        """Handle test command"""
+        container = self._make_project_container(
+            project_name=args.project, language=args.language if hasattr(args, "language") else None
+        )
+
+        if args.clear_cache:
+            for pattern in ["build", "build-*", "install"]:
+                for path in HoloHubCLI.HOLOHUB_ROOT.glob(pattern):
+                    if path.is_dir():
+                        if args.dryrun:
+                            print(f"  {Color.yellow('Would remove:')} {path}")
+                        else:
+                            shutil.rmtree(path)
+
+        container.dryrun = args.dryrun
+
+        container.build(
+            base_img=args.base_img,
+            build_args=args.build_args,
+        )
+
+        # Construct the ctest command line
+        # If we should run without xvfb
+        xvfb = "xvfb-run -a"
+        if args.no_xvfb:
+            xvfb = ""
+
+        base_img = args.base_img
+        if not base_img:
+            base_img = container.default_base_image()
```

Comment:
Nitpick, could simplify:

```suggestion
        base_img = args.base_img or container.default_base_image()
```